src/os/bluestore/BlueStore.cc

```diff
@@ -6783,19 +6767,20 @@ void BlueStore::_main_bdev_label_remove(Allocator* an_alloc)
 }
 
 int BlueStore::_check_or_set_bdev_label(
-  const string& path, uint64_t size, const string& desc, bool create)
+  const string& path, BlockDevice* bdev,
+  uint64_t size, const string& desc, bool create)
```

Comment:
size parameter is redundant now - can be obtained from bdev internally.

Reply:
In `_minimal_open_bluefs` we have a bit different code, that prints dout.
But maybe if we unify everything to `ceph_assert()` then maybe we could.